packages/iTwinUI-react/src/core/Table/Table.tsx

```diff
@@ -178,6 +179,11 @@ export type TableProps<
    * Must be memoized.
    */
   onFilter?: (filters: TableFilterValue<T>[], state: TableState<T>) => void;
+  /**
+   * Value used for global filtering.
+   * Use with `globalFilter` and/or `manualGlobalFilter` to handle filtering yourself e.g. filter in server-side.
+   */
+  globalFilterValue?: any; // eslint-disable-line @typescript-eslint/no-explicit-any
```

Comment:
Is there a reason you chose this name?
`globalFilter` might be more suitable for prop name.